common/lib/xmodule/xmodule/js/spec_util.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    'use strict';
+
+    jasmine.waitForInputAjax = function (conditionalFn) {
```

Comment:
I'm hoping to have the UI Toolkit installed in edx-platform by early next week, at which point that would be a great place to put it. It looks like it would make sense in here:

https://github.com/edx/edx-ui-toolkit/blob/master/src/js/utils/spec-helpers/ajax-helpers.js